### DISCUSSION TITLE
Consistent Syntax for functions in Docs

### DISCUSSION THREAD

**Discussion by dvd101x** [josdejong/mathjs#2938] [Ideas]
Hi, 

While reading about functions in the docs, I've noticed three different types of syntax.

- `math.func(...args)`
- `func(...args)`
- `X = math.func(...args)`

It might be better to have all of them with a similar syntax like `func(...args)`

https://mathjs.org/docs/reference/functions.html

**Comment by josdejong** [comment]
Agree, good point!

**Reply by dvd101x** [reply]
I made PR #2941 for this, just found and replaced so that everything is like `math.func(...args)` only a few were different.

**Comment by josdejong** [comment]
Fixed via #2941, closing this discussion now.
